app/scripts/modules/core/src/application/ApplicationDisplayRenderer.tsx

```diff
@@ -0,0 +1,36 @@
+import { IApplication } from 'core/domain';
```

Comment:
All the application renderer implementing classes have the same structure so if there's an issue with one, there's probably the same issue with all of them.